x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -65,22 +78,29 @@ xpack:
           pki1:
             username_pattern: "EMAILADDRESS=(.*?)(?:,|$)"
 ------------------------------------------------------------
+
+NOTE: If the regex is too restrictive and does not match the Subject DN of the
+client's certificate, then the realm will not authenticate the certificate.
```

Comment:
```suggestion
client's certificate, the realm cannot authenticate the certificate.
```

Reply:
I picked `client's certificate, the realm does not authenticate the certificate.`